chef/automate
# Fixing the way the project update precentage is calculated

### :nut_and_bolt: Description: What code changed, and why?

There was a bug in the way we were calculating the percentage complete for a project update. We were just taking the last value from all the domain services. This change cleans up the percentage complete function to remove the cereal code to make the code easier to read. 
 
### :chains: Related Resources
#2436 

### :+1: Definition of Done

The percentage complete does not show 100% when the time is in the future. 

### :athletic_shoe: How to Build and Test the Change
1. `build components/authz-service && start_all_services`
1. Add a lot of data to compliance with `chef_load_compliance_scans -N1000 -D10 -M2 -T20000`
1. Create a project
1. Apply the rules
1. Ensure the percentage complete is not always 100%

### :white_check_mark: Checklist
